### DISCUSSION TITLE
Instalação em Linux

### DISCUSSION THREAD

**Discussion by Aldrith72** [RamonSilva20/mapos#1637] [General]
Estou realizando uma instalação em Linux CentOS7, 
A instalação vai até o fim e informa realizada com sucesso !!!
Porém ao abrir o sistema aparece na tela a seguinte mensagem: Sua instalação está incompleta, por favor instale as dependências utilizando o composer. A pasta system não foi localizada.
A pasta system  não existe !!!!

**Comment by willph** [comment]
> Estou realizando uma instalação em Linux CentOS7, A instalação vai até o fim e informa realizada com sucesso !!! Porém ao abrir o sistema aparece na tela a seguinte mensagem: Sua instalação está incompleta, por favor instale as dependências utilizando o composer. A pasta system não foi localizada. A pasta system não existe !!!!

Faltou roda o comando no terminal, pasta raiz da instalação. 

`composer install --no-dev`

**Reply by Aldrith72** [reply]
Já foi realizado este procedimento, mas continua dando o mesmo erro

**Reply by Aldrith72** [reply]
Não criou a pasta System , ela deveria ter alguma coisa dentro ??? 
![mapos](https://user-images.githubusercontent.com/99360284/153278691-63f109a3-efb7-4863-a30e-71b0c843bbba.jpeg)

**Reply by willph** [reply]
> Não criou a pasta System , ela deveria ter alguma coisa dentro ??? ![mapos](https://user-images.githubusercontent.com/99360284/153278691-63f109a3-efb7-4863-a30e-71b0c843bbba.jpeg)

Veja se na pasta `application` tem uma pasta chamada `vendor`

**Comment by SuperPorte** [comment]
em distros debian like, eu rodo apt-get install *composer*
instala um monte de coisas pequenas, mas resolve....
no CentOS não sei como proceder

o Mint LMDE4 me facilita MUITO o dia-a-dia

**Reply by willph** [reply]
Ainda precisa da pasta?

**Comment by willph** [comment]
conseguiu?

**Comment by Aldrith72** [comment]
Já foi resolvido, consegui instalar a biblioteca.

Em sex., 11 de fev. de 2022 20:43, Wilmerson ***@***.***>
escreveu:
